### DISCUSSION TITLE
Getting member stars triggers http 418

### DISCUSSION THREAD

**Discussion by afkvido** [ProdigyErrorCodes/ProdigyErrorCodes#17] [New codes/triggers]
someone in the pmgh discord said that using WCM to get member stars triggers 418, can anyone replicate?

**Comment by donaldli2020** [comment]
i tested it out but the member stars just didn't work idk why

**Reply by afkvido** [reply]
did it giv u a 418?

**Reply by donaldli2020** [reply]
no, it just didn't work
i didn't get member stars

**Reply by donaldli2020** [reply]
go to @prodigyerrorcodes to join to organization (#15)

**Reply by afkvido** [reply]
sorry for the delay, I was asleep :\

I joined your organization :)

**Reply by donaldli2020** [reply]
asleep on 8 p.m. pst?

**Reply by afkvido** [reply]
> asleep on 8 p.m. pst?

it is 11:11pm pst

and I wasn't, I was just watching the mandalorian now that it costs literally nothing

**Comment by donaldli2020** [comment]
I can confrim
